src/controllers/timerController.js

```diff
@@ -1,4 +1,62 @@
+
+const logger = require('../startup/logger');
+const OldTimer = require('../models/oldTimer');
+
 const timerController = function (Timer) {
+  const getTimerFromDatabase = async ({ userId }) => {
+    try {
+      const timerObject = await Timer.findOne({ userId }).exec();
+      if (!timerObject) {
+          const newRecord = {
+            userId,
+            totalSeconds: 0,
+            isRunning: false,
+            isApplicationPaused: false,
+            isUserPaused: false,
+          };
+          const newTimer = await Timer.create(newRecord).exec();
```

Comment:
Good question, basically this is if there is no timer instance from the user in the database. Usually this could be since the user is new, there has been no timer created yet. 

`setTimerToDatabase` Updates a record, which assumes the timer already exist.